Do-Good-Get-Good/do-good-get-good
# Get user permission level from custom claims

This PR updates the way user level is fetched and handled.
It is now coming from the users custom claims instead of the Firestore object.

**I have also added a special `Developer Options` screen for `superusers` with the added `developer` permission**
Example of user with this permission: admin@dev.com

The only developer option currently available is one that copies all users permission level into each users custom claims.


### How to test
* Make sure you can login (test user, admin and superadmin accounts).
* Make sure the accounts still have different accesses.
If you're not able to log in to some accounts follow `step 1` under breaking changes.


## Breaking changes
1. Since these are breaking changes, please make sure you run the added developer option in `dev` before or directly after merging this PR!

2. To get this to work in `production` you have to upload the added cloud function `updateAllUsersPermissions` to the `production` backend otherwise you won't be able to run the developer option in the production app.
Since there are a few more steps to get this working in `production`, feel free to reach out to me and I'll help out.
